src/Microsoft.Health.Dicom.Client/IDicomWebClient.ExtendedQueryTag.cs

```diff
@@ -13,8 +13,13 @@ namespace Microsoft.Health.Dicom.Client
     public partial interface IDicomWebClient
     {
         Task<DicomWebResponse<OperationReference>> AddExtendedQueryTagAsync(IEnumerable<AddExtendedQueryTagEntry> tagEntries, CancellationToken cancellationToken = default);
+
         Task<DicomWebResponse> DeleteExtendedQueryTagAsync(string tagPath, CancellationToken cancellationToken = default);
+
         Task<DicomWebResponse<GetExtendedQueryTagEntry>> GetExtendedQueryTagAsync(string tagPath, CancellationToken cancellationToken = default);
-        Task<DicomWebResponse<IEnumerable<GetExtendedQueryTagEntry>>> GetExtendedQueryTagsAsync(CancellationToken cancellationToken = default);
+
+        Task<DicomWebResponse<IEnumerable<GetExtendedQueryTagEntry>>> GetExtendedQueryTagsAsync(int limit, int offset = 0, CancellationToken cancellationToken = default);
```

Comment:
nit: do we want to add default value on limit for the APIs easier to use?

Reply:
Yeah, that makes a lot of sense